azure/templates/tests.yml

```diff
@@ -25,7 +25,7 @@ steps:
               export BASE_URL="${{ parameters.base_url }}"
               export ROOT_PATH="$(SERVICE_BASE_PATH)"
               export CLIENT_ID="$(CLIENT_ID)"
-              export JWT_RS256_PRIVATE_KEY=$(echo "$(JWT_RS256_PRIVATE_KEY)" | base64 -d | sed -z 's/\n/\\n/g')
+              export TOKEN_URL=https://${{ parameters.environment }}.api.service.nhs.uk/oauth2/token
```

Comment:
See https://github.com/NHSDigital/ambulance-analytics-api/pull/117/files?file-filters%5B%5D=.json&file-filters%5B%5D=.xml&file-filters%5B%5D=.yml&hide-deleted-files=true#r578309598

Reply:
This one shouldn't cause any problem since we use `test.yml` for internal-dev and int. This is only a problem for smoke-test.yml when using prod